lib/ui/proposals/filter/interactor/filter_proposals_bloc.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:hypha_wallet/core/error_handler/error_handler_manager.dart';
+import 'package:hypha_wallet/core/error_handler/model/hypha_error.dart';
+import 'package:hypha_wallet/ui/architecture/interactor/page_states.dart';
+import 'package:hypha_wallet/ui/architecture/result/result.dart';
+import 'package:hypha_wallet/ui/profile/interactor/profile_data.dart';
+import 'package:hypha_wallet/ui/profile/usecases/fetch_profile_use_case.dart';
+import 'package:hypha_wallet/ui/proposals/filter/interactor/dao_proposal_count_entity.dart';
+import 'package:hypha_wallet/ui/proposals/filter/interactor/filter_status.dart';
+import 'package:hypha_wallet/ui/proposals/filter/usecases/aggregate_dao_proposal_counts_use_case.dart';
+import 'package:hypha_wallet/ui/proposals/filter/usecases/get_daos_from_proposal_counts_use_case.dart';
+import 'package:hypha_wallet/ui/proposals/list/interactor/proposals_bloc.dart';
+
+part 'page_command.dart';
+part 'filter_proposals_bloc.freezed.dart';
+part 'filter_proposals_event.dart';
+part 'filter_proposals_state.dart';
+
+class FilterProposalsBloc extends Bloc<FilterProposalsEvent, FilterProposalsState> {
+  final ProposalsBloc _proposalsBloc;
+  final FetchProfileUseCase _fetchProfileUseCase;
+  final AggregateDaoProposalCountsUseCase _aggregateDaoProposalCountsUseCase;
+  final GetDaosFromProposalCountsUseCase _getDaosFromProposalCountsUseCase;
+  final ErrorHandlerManager _errorHandlerManager;
+  late bool _isCalled;

```

Comment:
This flag is a little odd - what is this for, who is setting it, and why?

I don't see anything in this PR using this. 

If it has a use please rename to be more descriptive   

If not, delete ;)

Reply:
Yes you are right, I forgot to remove it.